openslides_backend/action/actions/assignment/create_update_delete.py

```diff
@@ -49,10 +49,28 @@ def update_instance(self, instance: dict[str, Any]) -> dict[str, Any]:
                     ["assignment_poll_add_candidates_to_list_of_speakers"],
                 ).get("assignment_poll_add_candidates_to_list_of_speakers")
             ):
+                list_of_speakers = self.datastore.get(
+                    fqid_from_collection_and_id(
+                        "list_of_speakers", assignment["list_of_speakers_id"]
+                    ),
+                    ["speaker_ids"],
+                )
+                pre_existing_speakers_meeting_user_ids = [
+                    pre_existing_speaker["meeting_user_id"]
+                    for pre_existing_speaker in self.datastore.get_many(
+                        [
+                            GetManyRequest(
+                                "speaker",
+                                list_of_speakers["speaker_ids"],
```

Comment:
Are you sure, that los always contains a speaker_ids and that it is not empty? Please write a test with an empty los and try it.

Reply:
Good point. I fixed that.